conf/workloads/graph/pagerank.conf

```diff
@@ -0,0 +1,35 @@
+#Pagerank
+hibench.pagerank.tiny.pages			50
+hibench.pagerank.tiny.num_iterations		1
+hibench.pagerank.tiny.block			0
+hibench.pagerank.tiny.block_width		16
+hibench.pagerank.small.pages			5000
+hibench.pagerank.small.num_iterations		3
+hibench.pagerank.small.block			0
+hibench.pagerank.small.block_width		16
+hibench.pagerank.large.pages			500000
+hibench.pagerank.large.num_iterations		3
+hibench.pagerank.large.block			0
+hibench.pagerank.large.block_width		16
+hibench.pagerank.huge.pages			5000000
+hibench.pagerank.huge.num_iterations		3
+hibench.pagerank.huge.block			0
+hibench.pagerank.huge.block_width		16
+hibench.pagerank.gigantic.pages			30000000
+hibench.pagerank.gigantic.num_iterations	3
+hibench.pagerank.gigantic.block			0
+hibench.pagerank.gigantic.block_width		16
+hibench.pagerank.bigdata.pages			50000000
+hibench.pagerank.bigdata.num_iterations		3
+hibench.pagerank.bigdata.block			0
+hibench.pagerank.bigdata.block_width		16
```

Comment:
If the two PageRank(RDD-based & GraphX-based) shared the same set of parameters, we only need to keep one, instead of keeping 2 same pagerank.conf under `graph/` and `websearch/` ?

Reply:
It needs to be seperated for misuse. And user may need to benchmark the two at the same time with different pararms.

Reply:
If they need to benchmark them at the same time, shouldn't we differentiate the parameters name here? e.g. `hibench.graph.pagerank.xx`

Reply:
I think it's OK . Since the config path clearly shows the difference, to be consistent with others. We may just need to document it.